system-tests/tests/smoke/cre/environment-one-don-read-contract.toml

```diff
@@ -0,0 +1,77 @@
+
+[blockchain_a]
+  type = "anvil"
+  chain_id = "1337"
+
+[custom_anvil_miner]
+  block_speed_seconds = 5
+
+[jd]
+  # change to your version
+  image = "job-distributor:0.9.0"
+
+# fake data provider used as a mocked price source
+[fake]
+  port = 8171
```

Comment:
remove? we don't really need this port, because we are not starting a mock http server